client/webserver/site/src/localized_html/en-US/wallets.tmpl

```diff
@@ -220,39 +313,37 @@
         </div>
       </div>
 
-      <div id="showChangeType" class="py-1 mt-2 pointer d-flex align-items-center justify-content-start">
+      <div id="showChangeType" class="my-2 pointer d-flex align-items-center justify-content-start">
         <span id="changeTypeShowIcon" class="ico-plus fs8 ps-1"></span>
         <span id="changeTypeHideIcon" class="ico-minus fs8 ps-1 d-hide"></span>
         <span id="changeTypeMsg" class="d-inline-block ps-1 pb-1">change the wallet type</span>
       </div>
-      <div id="changeWalletType" class="py-1 d-hide">
+      <div id="changeWalletType" class="d-hide mb-3">
+        <div class="my-2">
+          Note: Changing to a different wallet while having active trades might cause funds to be lost.
+        </div>
         <select id="changeWalletTypeSelect"></select>
       </div>
-      <div class="my-2">
-        Note: Changing to a different wallet while having active trades might cause funds to be lost.
-      </div>
       <div class="d-flex mt-1 {{if $passwordIsCached}}justify-content-end{{end}}">
         <div class="col-12 p-0 {{if $passwordIsCached}}d-hide{{end}}">
           <label for="appPW" class="form-label ps-1 mb-1">App Password</label>
           <input type="password" class="form-control select" id="appPW" autocomplete="off">
         </div>
         <div class="col-12 p-0 text-end">
           <div>&nbsp;</div>
-          <button id="submitReconfig" type="button" class="w-75 mt-1 justify-content-center fs15 bg2 selected">Submit</button>
+          <button id="submitReconfig" type="button" class="w-75 mb-1 justify-content-center fs15 bg2 selected">Submit</button>
         </div>
       </div>
       <div class="fs15 pt-3 text-center d-hide errcolor text-break" id="reconfigErr"></div>
       <hr class="dashed my-2">
-      <div class="d-flex flex-row">
-        <button id="downloadLogs" type="button" class="w-25 mt-1 mx-1 justify-content-center fs15 bg2">Wallet Logs</button>
+      <div class="sans-light fs17">Other Actions</div>
+      <div class="d-flex flex-row fs15 pt-1">
+        <button id="downloadLogs" type="button" class="p-1 me-2 hoverbg">Wallet Logs</button>
         <button id="exportWallet" type="button" class="w-25 mx-1 mt-1 justify-content-center fs15 bg2">Export Wallet</button>
-        <button id="recoverWallet" type="button" class="danger w-25 mx-1 mt-1 justify-content-center fs15 bg2">Recover</button>
+        <button id="recoverWallet" type="button" class="danger p-1 me-2 hoverbg">Recover</button>
+        <button id="rescanWallet" type="button" class="danger p-1 me-2 hoverbg">Rescan</button>
```

Comment:
When I click Rescan, there's no indication that it's rescanning.  Maybe close the reconfig form, sending them back to see the sync status icon on the wallets list?

Reply:
I'm seeing a Rescan button for wallets that don't support rescanning.